samples/Exporters/Console/TestRedis.cs

```diff
@@ -28,8 +27,17 @@ internal class TestRedis
     {
         internal static object Run(string zipkinUri)
         {
-            // connect to the server
-            var connection = ConnectionMultiplexer.Connect("localhost:6379");
+            /*
```

Comment:
https://github.com/open-telemetry/opentelemetry-dotnet/pull/799/files#diff-23d4b7cb5b570c90d19a18ed11a9e347R41

can you get this too into program.cs?

Reply:
i will push soon